showcase.json

```diff
@@ -2811,6 +2811,26 @@
     "youtube": "https://www.youtube.com/channel/UCdMLfGmuLAsQlVPg9QROfcQ",
     "submittableId": "32119235"
   },
+  {
+    "name": "Sama Network",
+    "description": "SAMA network, with its original designed consensus and business sharding separation, builds a highly secure and private distributed computational system. It provides data security and computational power for the World Wide Web without altering users’ habits.",
+    "website": "https://sama.network/",
+    "logo": "/img/showcase/sama_network_logo.png",
+    "screenshots": [],
+    "video": "",
```

Comment:
Can you please remove the video key value if you do not have a video?

Reply:
OK,got it.